rinex/db/NAV/orbits.json

```diff
@@ -680,20 +688,22 @@
             "idot": "f64",
             "deltaN0Dot": "f64",
             "satType": "f64",
+            "t_op": "f64",
             "sisa1_oe": "f64",
             "sisa1_ocb": "f64",
             "sisa1_oc1": "f64",
             "sisa1_oc2": "f64",
-            "isc_b1cd": "f64",
-            "tgd_b1cp": "f64",
-            "tgd_b2ap": "f64",
+            "iscB1Cd": "f64",
+            "spare1": "xxx",
+            "tgdB1Cp": "f64",
```

Comment:
I changed the naming style of the BDS stuff to be similar to GPS/QZSS, one advantage of camel case is that I can correctly mimic the capital letters in the signal name which has special meaning and matches BDS ICDs

Reply:
good idea